runtime/queries/rust/highlights.scm

```diff
@@ -320,6 +320,7 @@
   "'"
 ] @operator
 
+((try_expression "?") @operator)
```

Comment:
```suggestion
(try_expression "?" @operator)
```

so that the capture applies only to the `?` and not the overall try_expression node